4-forms-validations/form.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formulario de validación</title>
+</head>
+<body>
+
+    <form action="">
+        <label for="name">Nombre</label>
+        <input type="text" id="name" name="user_name">
+        <label for="name">Apellido</label>
+        <input type="text" id="apellido" name="user_name"> <br>
+        <label for="mail">Correo Electrónico</label>
+        <input type="email" id="mail" name="user_mail" >
+        <label for="date"> fecha</label>
+        <input type="date" id="start" name="trip-start"
+         value="2018-07-22"
+          min="2018-01-01" max="2022-12-31"><br>
+        <input type="radio" name="Hombre" value="Hombre"> Hombre
+
+        <input type="radio" name="Mujer" value="Mujer"> Mujer <br>
```

Comment:
```suggestion
  <select name="gender" id="gender">
  <option value="m">Masculino</option>
 <option value="f">Femenino</option>
  <option value="o">Otro</option>
    </select>
```
En la actividad se especifica usar un `<select>` para el caso del género.